### PR TITLE
fix: add more language callbacks

### PR DESCRIPTION
**Pull request by markusahlstrand**
Think we need to reconsider the language handling :) Moving to liquid would solve it but maybe we need another solution first as this will get out of hand

### CODE REVIEW

**Comment by danielgent** [comment]
Moving to liquid would only solve it in the case of it being the same domain, so let's just whitelist login2.sesamy.dev/com

What's the security risk there?

File: src/utils/validate-redirect-url.ts [hunk 1]
```diff
@@ -12,26 +12,36 @@ const ALLOWED_CALLBACK_URLS = [
...
-  "https://login2.sesamy.dev/enter-code",
   "https://login2.sesamy.dev/sv/",
+  "https://login2.sesamy.dev/en/",
+  "https://login2.sesamy.dev/enter-code",
+  "https://login2.sesamy.dev/sv/enter-code",
+  "https://login2.sesamy.dev/en/enter-code",
   "https://login2.sesamy.dev/callback",
   "https://login2.sesamy.dev/sv/callback",
-  "https://login2.sesamy.dev/sv/enter-code",
+  "https://login2.sesamy.dev/en/callback",
```
Comment (danielgent) [inline src/utils/validate-redirect-url.ts:22]: where is this coming from though?

There should never be an `/en`

This is simply replicating Auth0. I think we can just white list everything on login2

*BUT* we have too many URLs in here from reading the path in the host for allowed origins

**Comment by markusahlstrand** [comment]
Yes, whitelisting the entire login domain sounds like a plan..

File: src/utils/validate-redirect-url.ts (hunk 1 above)
Comment (danielgent) [reply src/utils/validate-redirect-url.ts:22]: wow https://github.com/vercel/next.js/issues/21040

It didn't use to be like this?

**Pull request merged**